dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/user/MeController.java

```diff
@@ -387,6 +387,10 @@ public void changePassword( @RequestBody Map<String, String> body, HttpServletRe
         manager.update( currentUser );
 
         currentUserService.expireUserSessions();
+
+        String logoutUrl = response.encodeRedirectURL( "/dhis-web-commons-security/logout.action" );
```

Comment:
Should we move this to a Java constant somewhere such as the SecurityService? These web URLs are prone to be forgotten and break stuff when we change the config.